docker-compose.yml

```diff
@@ -4,9 +4,10 @@ services:
     command: bash -c "yarn install && yarn test"
     build:
       context: .
-      target: puppeteer
+      target: CI
```

Comment:
How changes to this file are related to this pr?

Reply:
We run the screenshots specs in a docker image, this change was needed for the image to run...